### PR TITLE
Feature/add animetorrents

### PR DESCRIPTION
**Pull request by p0psicles**
- [x] PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [x] Read the [contribution guide](https://github.com/pymedusa/Medusa/blob/master/.github/CONTRIBUTING.md)

- [x] Add season pack support. Thought of it later, probably will need to make an adjustment to the season search string.

### CODE REVIEW

**Comment by p0psicles** [comment]
This provider works now. But because of the 'strange' release names from this provider, we still need to make some changes to our name/title parser, before it will become really useful.

Created an issue here: #1747

File: medusa/providers/torrent/html/animetorrents.py [hunk 1]
```diff
@@ -245,9 +245,16 @@ def _get_episode_search_strings(self, episode, add_string=''):
             'Episode': []
         }
 
+        season_scene_names = scene_exceptions.get_scene_exceptions(episode.show.indexerid, season=episode.scene_season)
```
Comment (p0psicles) [inline medusa/providers/torrent/html/animetorrents.py]: I'd like to implement something like this in genericprovider. But will possibly modify (or create new) allPossibleShowNames. As there get_scene_exceptions is already called. Just as well return it with the matching season if available.

**Comment by medariox** [comment]
Approved 👍

File: medusa/providers/torrent/html/animetorrents.py [hunk 2]
```diff
@@ -47,26 +47,27 @@ def __init__(self):
...
         self.anime_only = True
-        self.categories = {2: 'Anime Series',
-                           7: 'Anime Series HD'}
+        self.categories = {
+            2: 'Anime Series',
+            7: 'Anime Series HD'
+        }
 
         # Proper Strings
+        self.proper_strings = []
```
Comment (duramato) [inline medusa/providers/torrent/html/animetorrents.py]: Are there no propers? or alias tags like v1/v2?

**Pull request merged**